### DISCUSSION TITLE
Update to Greek Translation

### DISCUSSION THREAD

**Discussion by fan13pl** [aclap-dev/video-downloadhelper#970] [Language & Translation]
[el.txt](https://github.com/user-attachments/files/16103353/el.txt)

**Comment by paulrouget** [comment]
Updated! Should be in v9.1 Thanks :)

**Comment by fan13pl** [comment]
No problem, happy to help!

Στις Τρί 16 Ιουλ 2024, 10:14 ο χρήστης Paul ***@***.***>
έγραψε:
